repos/system_upgrade/common/actors/checkconsumedassets/libraries/check_consumed_assets.py

```diff
@@ -101,13 +121,16 @@ def report_malformed_assets(malformed_assets):
             details = ' - The asset file {filename} contains invalid value in its "{data_streams_field}"'
             details = details.format(filename=asset.filename, data_streams_field=ASSET_PROVIDED_DATA_STREAMS_FIELD)
         summary_lines.append(details)
-        docs_url_to_title_map[asset.docs_url].append(asset.docs_title)
+        if asset.docs_url:
+            # Add URLs only when they are specified. docs_url could be empty string
+            docs_url_to_title_map[asset.docs_url].append(asset.docs_title)
 
     report_parts = [
         reporting.Title(title),
         reporting.Summary('\n'.join(summary_lines)),
+        reporting.Remediation(hint=_get_hint()),
         reporting.Severity(reporting.Severity.HIGH),
-        reporting.Groups([reporting.Groups.INHIBITOR, reporting.Groups.REPOSITORY]),
+        reporting.Groups([reporting.Groups.INHIBITOR, reporting.Groups.SANITY]),
     ]
 
     report_parts += make_report_entries_with_unique_urls(docs_url_to_title_map)
```

Comment:
discussed via slack that this would lead to a unexpected nested list inside `report_parts` of having a flat list. Both agreed to keep it rather unchanged.